pkg/openwsn/include/openwsn.h

```diff
@@ -33,6 +33,15 @@ extern "C" {
 #define  OPENWSN_PANID  (0xCAFE)
 #endif
 
+/**
+ * @brief   Default NID for OpenWSN network
+ *
+ * @note Only one netif is currently possible in OpenWSN.
+ */
+#ifndef  OPENWSN_NETIF_ID
+#define  OPENWSN_NETIF_ID  (0x0001)
+#endif
```

Comment:
```suggestion
#ifndef  CONFIG_OPENWSN_NETIF_ID
#define  CONFIG_OPENWSN_NETIF_ID   (0x0001)
#endif
```

For future `Kconfig` compatibility.

Reply:
addressed